lib/console.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Console do
+  def get_user_input(message) do
+    String.trim(IO.gets(message))
+  end
+
+  def display_message(message) do
+    IO.puts(message)
+  end
+
+  def display_cond_message(user_answer, yes_msg, no_msg) do
+    if user_answer == true do
```

Comment:
Here you can use pattern matching instead of `if`, `else` statements. Something that you can look into could be defining multiple function heads.

Reply:
Actually looking back on this, there are some repetition with `display_message` & `display_cond_message`.

Reply:
Yes - I will completely delete `display_cond_message`, agreed that the console shouldn't be performing any logic, - it should just print whatever message it receives. (I should know better 😳)